sbnanalysis/CMakeLists.txt

```diff
@@ -15,6 +14,8 @@ include(${ROOT_USE_FILE})
 
 include_directories(${PROJECT_SOURCE_DIR})
 
+file(MAKE_DIRECTORY ${CMAKE_INSTALL_PREFIX}/fcl)
```

Comment:
A very necessary directory, I made it too in https://cdcvs.fnal.gov/redmine/projects/sbncode/repository/revisions/197a57ad1912e07eda9ef3e0e815b27dc43db776/diff! 😄

Reply:
oh that's funny! Should I remove this line then?